test/functional/internal/test_utils.py

```diff
@@ -0,0 +1,119 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+"""Test suite for ``dynamodb_encryption_sdk.internal.utils``."""
+import copy
+
+import pytest
+
+from dynamodb_encryption_sdk.encrypted import CryptoConfig
+from dynamodb_encryption_sdk.identifiers import CryptoAction
+from dynamodb_encryption_sdk.internal.utils import encrypt_batch_write_item
+from dynamodb_encryption_sdk.material_providers import CryptographicMaterialsProvider
+from dynamodb_encryption_sdk.structures import AttributeActions, EncryptionContext
+from dynamodb_encryption_sdk.transform import ddb_to_dict
+
+from ..functional_test_vector_generators import attribute_test_vectors
+
+
+def get_test_item(standard_dict_format, partition_key, sort_key):
+    attributes = attribute_test_vectors("serialize")
+
+    attributes = {"attr_" + str(pos): attribute[0] for pos, attribute in enumerate(attributes)}
+    attributes["partition-key"] = {"S": partition_key}
+    if sort_key:
```

Comment:
Minor: For comparing to `None`, we should be explicit.

http://docs.quantifiedcode.com/python-anti-patterns/readability/comparison_to_none.html